pelicun/uq.py

```diff
@@ -696,25 +683,24 @@ def _neg_log_likelihood(
     likelihoods = np.clip(likelihoods, a_min=np.nextafter(0, 1), a_max=None)
 
     # calculate the total negative log likelihood
-    NLL = -(
+    null = -(
```

Comment:
`null` is definitely not a good idea for naming this variable. The original NLL stood for negative log likelihood. I can see that using NLL is not ideal, so how about `negative_log_likelihood` instead?

Reply:
Sorry about that, I definitely meant to type nll, lowercase NLL. But negative_log_likelihood is better. 07dc45fc